Sources/BraintreeCore/Configuration/ConfigurationLoader.swift

```diff
@@ -43,21 +41,17 @@ class ConfigurationLoader {
     @_documentation(visibility: private)
     @ConfigurationActor
     func getConfig() async throws -> BTConfiguration {
+        var existingTask: Task<BTConfiguration, Error>
```

Comment:
I just noticed this commit with moving existingTask as property in this function. Would this eliminate benefits of deduplication of the task in quick successive calls?